node/src/database/rocksdb.rs

```diff
@@ -446,14 +469,21 @@ impl<'db, DB: DBAccess> Ledger for DBTransaction<'db, DB> {
         }
     }
 
-    fn fetch_block_header(
-        &self,
-        hash: &[u8],
-    ) -> Result<Option<(ledger::Header, Vec<[u8; 32]>)>> {
+    fn fetch_block_light(&self, hash: &[u8]) -> Result<Option<HeaderRecord>> {
```

Comment:
Not sure this is the most appropriate name...
Since we're return a HeaderRecord, I'd either call it `fetch_header_record` or, to be clearer, `fetch_extended_header`.

Reply:
As discussed, I renamed HeaderRecord into LightBlock